bcdaworker/queueing/worker_cleanup.go

```diff
@@ -48,14 +48,9 @@ func (w *CleanupJobWorker) Work(ctx context.Context, rjob *river.Job[worker_type
 	stagingDir := conf.GetEnv("FHIR_STAGING_DIR")
 	payloadDir := conf.GetEnv("FHIR_PAYLOAD_DIR")
 	environment := conf.GetEnv("DEPLOYMENT_TARGET")
+	slackToken := conf.GetEnv("SLACK_TOKEN")
```

Comment:
Does this PR also need new SOPS entries for the new env vars (and/or an update to bcda-ops)?